ui/packages/consul-ui/app/components/hashicorp-consul/index.hbs

```diff
@@ -83,16 +83,23 @@
     <a
       href={{href-to 'index' params=(hash peer=undefined)}}
     ><Consul::Logo /></a>
+    {{#if (eq @dcs.length 1)}}
+      <div class="dc-name" data-test-datacenter>
+        {{@dc.Name}}
+      </div>
+    {{/if}}
```

Comment:
What would you think of moving this logic in the `Datacenter::Selector` component? I know we aren't giving the user the ability to 'select' a datacenter as they only have one, but the logic essentially 'selects' that single datacenter for the user as they only have one, which makes sense to me.

Also if we ever feature flag out datacenters entirely (not sure if this will ever happen, but it could), then this entire `Datacenter::Selector` component can use out wiring/injection approach to feature flagging, as everything to do with showing/selecting your selected datacenter is all self-contained in the one component we can just inject in/out different implementations.

This is also pretty much how the Partition::Selector works:

https://github.com/hashicorp/consul/blob/610038ce670213055da323ecfc4f9cf06f4f430b/ui/packages/consul-partitions/app/components/consul/partition/selector/index.hbs#L71-L77